### PR TITLE
Updated healthcheck attribute to use valid types

### PR DESCRIPTION
**Pull request by ktmitton**
TimeSpan? and int? are not valid for attributes, so updated to use valid types that can be resolved at compile time.

### CODE REVIEW

File: Mittons.Fixtures/Docker/Attributes/HealthCheckAttribute.cs [hunk 1]
```diff
@@ -11,13 +11,13 @@ public class HealthCheckAttribute : Attribute, IOptionAttribute
 
         public string Command { get; set; }
 
-        public TimeSpan? Interval { get; set; }
+        public int Interval { get; set; }
```
Comment (Frueber) [inline Mittons.Fixtures/Docker/Attributes/HealthCheckAttribute.cs:14]: Any benefits in using unsigned int for these? Seeing that they're only utilized if they're greater than 0.

**Review by Frueber — COMMENTED** [review]
Would we also want to consider unit tests for 0 and negative values? What do we expect the outcome to be for them?

File: Mittons.Fixtures/Docker/Attributes/HealthCheckAttribute.cs (hunk 1 above)
Comment (ktmitton) [reply Mittons.Fixtures/Docker/Attributes/HealthCheckAttribute.cs]: That's a good point...

I actually changed these to bytes, so we're getting unsigned, but we're also limiting size. Given the nature of these flags, if you need something larger than a byte, you likely are doing something wrong. If your service can't spin up within 18 hours, you have a big problem...

**Comment by ktmitton** [comment]
> Would we also want to consider unit tests for 0 and negative values? What do we expect the outcome to be for them?

based on your other suggest, we don't have to deal with negative, but there is value in checking 0 or not setting them (so defaults to 0). I'll get those added!

**Comment by ktmitton** [comment]
> > Would we also want to consider unit tests for 0 and negative values? What do we expect the outcome to be for them?
> 
> based on your other suggest, we don't have to deal with negative, but there is value in checking 0 or not setting them (so defaults to 0). I'll get those added!

@Frueber ok, added 3 new tests to support a couple missed variations.

**Review by Frueber — APPROVED** [review]
